CHANGES.rst

```diff
@@ -313,6 +313,9 @@ New Features
   - Add bolometric absolute and apparent magnitudes, ``M_bol`` and ``m_bol``.
     [#4262]
 
+  - Ensured that with ``spectral_density`` equivalency one could also convert
```

Comment:
You marked the PR as a bug fix but the entry is under "new features". Is this intentional?